src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/ManagingOfficerAdditionServiceTest.java

```diff
@@ -167,4 +175,88 @@ private void assertCorporateManagingOfficerDetails(CorporateManagingOfficerAddit
         assertEquals("Register name", corporateManagingOfficerAddition.getCompanyIdentification().getRegisterLocation());
         assertEquals("Registration number", corporateManagingOfficerAddition.getCompanyIdentification().getRegistrationNumber());
     }
+
+    @Test
+    void testCorporateBeneficialOwnerAdditionsAddressSameAsFlagTrue() {
```

Comment:
Change these test names to say managing officers instead of beneficial owners

Reply:
That was stupid of me 

This has now been resolved